test/health_cards/health_card_test.rb

```diff
@@ -21,6 +21,10 @@ class HealthCardTest < ActiveSupport::TestCase
     assert @health_card.bundle.is_a?(FHIR::Bundle)
   end
 
+  # test "metadata walking" do
+  #   byebug
+  # end
```

Comment:
Do we want to keep this?

Reply:
Oops, I was originally going to write new tests but decided to just make it pass the existing ones to start. I'll remove this